User_Story#1/SortingApp/SortinfApp.UnitTests/Utilities/DataWriterTest.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.IO;
+using BusinessLayer.Reader;
+using BusinessLayer.Utilities.Validator.Interfaces.Fakes;
+using BusinessLayer.Writer;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SortinfApp.UnitTests.Utilities
+{
+    /// <summary>
+    /// Represents tests for DataWriter class.
+    /// </summary>
+    /// <owner>Anton Petrenko</owner>
+    [TestClass]
+    public class DataWriterTest
+    {
+        /// <summary>
+        /// Tests WriteContent if pass empty collection it will throw ArgumentNullException.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        [TestMethod]
+        public void DataWriter_WriteContent_PassEmptyCollection_ThrowArgumentNullException()
```

Comment:
You are checking if collection is null, not empty.
Please don't be confused with these terms.

Reply:
One more test method need to be added to check if collection is empty.